include/networkit/centrality/DynApproxBetweenness.hpp

```diff
@@ -25,61 +24,64 @@ namespace NetworKit {
  * @ingroup centrality
  * Interface for dynamic approximated betweenness centrality algorithm.
  */
-class DynApproxBetweenness: public Centrality, public DynAlgorithm {
+class DynApproxBetweenness : public Centrality, public DynAlgorithm {
 
 public:
     /**
-      * The algorithm approximates the betweenness of all vertices so that the scores are
-      * within an additive error @a epsilon with probability at least (1- @a delta).
-      * The values are normalized by default.
-      *
-      * @param	G			the graph
-      * @param	storePredecessors   keep track of the lists of predecessors?
-      * @param	epsilon		maximum additive error
-      * @param	delta		probability that the values are within the error guarantee
-      * @param	universalConstant	the universal constant to be used in
-      * computing the sample size. It is 1 by default. Some references suggest
-      * using 0.5, but there is no guarantee in this case.
+     * The algorithm approximates the betweenness of all vertices so that the scores are
+     * within an additive error @a epsilon with probability at least (1- @a delta).
+     * The values are normalized by default.
+     *
+     * @param	G			the graph
+     * @param  storePredecessors   keep track of the lists of predecessors?
+     * @param	epsilon		maximum additive error
+     * @param	delta		probability that the values are within the error guarantee
      */
     DynApproxBetweenness(const Graph &G, double epsilon = 0.01, double delta = 0.1,
-                         bool storePredecessors = true, double universalConstant = 1.0);
+                         bool storePredecessors = true, double universalConstant = 0.5);
```

Comment:
The documentation for `universalConstant` should not be removed. Also, why the default value needs to be changed? According to the original documentation, it should be 1.

Reply:
Because the reference paper uses 0.5 and I have no idea where the 1.0 in the original documentation came from as all references I found to the universal constant have been 0.5.